MIPX/MIPX.md

```diff
@@ -20,59 +20,67 @@ Replaces: N/A
+**MIPXc4: Contested Modification of Management**  
+MIPXc4 is a process component which allows authors to appeal to Maker Governance to resolve a contested moficiation of management of one or more Intangible Assets owned by MakerDAO.
 
 ## Motivation
 
-The goal of MIPX is to provide a lightweight and efficient mechanism for the modification of control of any digital assets owned by MakerDAO. I have attempted to meet that goal by only requiring an on-chain vote in the event of disagreement between party's over which should have control of a digital asset.
+The goal of MIPX is to provide a lightweight and efficient mechanism for the Modification of Management of any Intangible Assets owned by MakerDAO. I have attempted to meet that goal by only requiring an on-chain vote in the event of disagreement between party's over which should have control of a Intangible Asset.
 
-A secondary motivation of MIPX is to create a publicly accessible asset registry that lists the key information about digital assets owned by MakerDAO.
+A secondary motivation of MIPX is to create a publicly accessible asset registry that lists the key information about Intangible Assets owned by MakerDAO.
 
 ## Specification / Proposal Details
 
-### Definitions
+### MIPXc1: Definitions and Assumptions**  
 
-Agreed Modification of Control: An uncontested and mutually agreed-upon transfer of control of a digital asset owned by MakerDAO.
+The following definitions are used within MIPX to describe the processes and actors involved in the control and assignment of Intangible Assets.
 
-Contested Modification of Control: A Process MIP component that allows for the resolution of Contested control over a digital asset by appealing to Maker Governance.
+#### Processes
+**Agreed Modification of Management:** An uncontested and mutually agreed-upon transfer of management control of a Intangible Asset owned by MakerDAO. A subproposal is not required for an agreed Modification of Management.
 
-Owner-Controller: An individual or organization that's been granted access to an asset held by the Dai Foundation.
+**Contested Modification of Management:** A subproposal that enables the resolution of a contested transfer of management control of a Intangible Asset by appealing to Maker Governance.
 
-### MIPXc1: Assumptions**  
-MIPX assumes a few things regarding control and ownership of digital assets within MakerDAO.
+#### Actors
+**Asset Owner:** An individual or organization that retains control of an asset on behalf of MakerDAO. An Asset Owner has the power to forcably revoke access and control of a Intangible Asset from a Content Manager.An Asset Owner has the power to forcably revoke access and control of a digitial asset from a Platform Manager when platform is not managed outside of the control of MakerDAO (twitter, github, etc.) The Dai Foundation is an example of an Asset Owner.
```

Comment:
Add a space between "...Content Manager.An Asset Owner"
Change "forcably" to "forcibly" and "digitial" to "digital"
Add a period after "(twitter, github, etc.)"